frontend/assets/javascripts/src/modules/paidToPaid.es6

```diff
@@ -41,27 +41,28 @@ export function init() {
         ajax({
             type: 'json',
             url: window.location.pathname.replace('change', 'preview'),
-            data: {promoCode: PROMO_CODE.val()}
-        }).then(result => {
-            LOADER.removeClass('is-loading');
-
-            if (result.error) {
-                FEEDBACK_CONTAINER.html(promoError({errorMessage: result.error}));
-                return;
+            data: {promoCode: PROMO_CODE.val()},
```

Comment:
@paulbrown1982 - is this promo code stuff still needed?

Reply:
Not in membership